src/care_radiology/api/webhooks.py

```
@@ -114,3 +135,173 @@ def save_webhook(self, request):
+            if not tag_config:
+                logger.error(f"[MPPS] Tag not found for status: {mpps_status}")
+                return Response(
+                    {
+                        "detail": f"Tag configuration not found for status: {mpps_status}"
+                    },
+                    status=status.HTTP_400_BAD_REQUEST,
+                )
+
+            tag_uuid = tag_config.external_id
+            logger.info(f"[MPPS] Tag found: {tag_uuid}")
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)